### PR TITLE
4.1.2rc

### PR DESCRIPTION
**Pull request by jngrad**
4.1.2 bugfix release. See [release notes](https://github.com/espressomd/espresso/wiki/es-412-release-notes).

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -2,6 +2,97 @@
...
+
+* Remove correlation between the rotational noise and translational
+  noise in the Langevin thermostat (#3355)
+
+* Fix a bug that may cause the wrong temperature to be set by the
+  Langevin thermostat in the first time step after the system was
+  altered from the Python level, e.g., by changing particles or
+  interactions (#3341)
+
+* Fix a bug that caused the DPD thermostat to be inactive when used
```
Comment (KaiSzuttor) [inline NEWS]: it was not inactive it resulted in the wrong velocity distribution

File: NEWS [hunk 2]
```diff
@@ -2,6 +2,97 @@
...
+Below, please find the list of changes. The numbers in brackets refer to
+ticket numbers on https://github.com/espressomd/espresso
+
+General corrections and improvements:
+
+* Remove correlation between the rotational noise and translational
+  noise in the Langevin thermostat (#3355)
+
+* Fix a bug that may cause the wrong temperature to be set by the
+  Langevin thermostat in the first time step after the system was
```
Comment (KaiSzuttor) [inline NEWS]: \+ DPD thermostat

**Comment by RudolfWeeber** [comment]
Looks good, except that project version needs to be changed in CMakeList.txt and maybe also in src/config/version.hpp.in Once that's done, it's ready

**Comment by jngrad** [comment]
@junghans: merged, tagged and released. The official announcement will me made shortly.
https://github.com/espressomd/espresso/releases/tag/4.1.2

**Comment by junghans** [comment]
@jngrad works:
https://bodhi.fedoraproject.org/updates/FEDORA-2019-5c823ff54b
https://bodhi.fedoraproject.org/updates/FEDORA-2019-41c0b986dc

there is still something wrong on EPEL8 though:
https://koji.fedoraproject.org/koji/taskinfo?taskID=39543181
x86_64 and ppc64le, uxc failure:
```
[buildvm-20:1302195:0:1302195] Caught signal 11 (Segmentation fault: invalid permissions for mapped object at address 0x7f64cb130768)
```
aarch64:
```
ImportError: /lib64/libgomp.so.1: cannot allocate memory in static TLS block
```

**Comment by jngrad** [comment]
@junghans Thanks!
> `[buildvm-20:1302195:0:1302195] Caught signal 11 (Segmentation fault: invalid permissions for mapped object at address 0x7f64cb130768)`

This UCX-related failure seems identical to the one you previously reported in #3307. According to @opoplawski the bug seems to have been fixed in UCX 1.5.2 (the build log shows UCX 1.4.0).

> `ImportError: /lib64/libgomp.so.1: cannot allocate memory in static TLS block`

I think I saw this one too in the RHEL8 build on Copr. Maybe related to [Bugzilla 1722181](https://bugzilla.redhat.com/show_bug.cgi?id=1722181).

**Pull request merged**